sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -2540,6 +2540,17 @@ class DataFrameSuite extends QueryTest
       assert(df.schema === new StructType().add(StructField("d", DecimalType(38, 0))))
     }
   }
+
+  test("SPARK-32356: forbid null type in create view") {
```

Comment:
can we put test in one place `SQLViewSuite`?

Reply:
moved this